### ISSUE TITLE
Amazon EC2 endpoint url different for eu-central-1

### ISSUE THREAD

**Issue by vissree** [chef-cookbooks/aws#101]
Got the following error when trying to use aws cookbook in ec2-central-1 region.
```
ec2-54-93-80-34.eu-central-1.compute.amazonaws.com [2015-01-21T12:12:19+00:00] WARN: Rightscale::HttpConnection : re-raising same error: https://eu-central-1.ec2.amazonaws.com:443 temporarily unavailable: (SocketError: getaddrinfo: Name or service not known) -- error count: 4, error age: 0
```

Looks like the endpoint is different for eu-central-1 vs other regions.
```
$ host ec2.eu-central-1.amazonaws.com
ec2.eu-central-1.amazonaws.com has address 54.239.54.44

$ host eu-central-1.ec2.amazonaws.com
Host eu-central-1.ec2.amazonaws.com not found: 3(NXDOMAIN)

$ host eu-west-1.ec2.amazonaws.com
eu-west-1.ec2.amazonaws.com has address 178.236.6.52

$ host ec2.eu-west-1.amazonaws.com
ec2.eu-west-1.amazonaws.com is an alias for eu-west-1.ec2.amazonaws.com.
eu-west-1.ec2.amazonaws.com has address 178.236.4.54
```

**Comment by vervas** [comment]
experiencing this issue too

**Comment by schlitzered** [comment]
same here

**Comment by martinfobian** [comment]
Same issue. Any workarounds anyone can offer?

**Comment by tas50** [comment]
This should be resolved in the 2.8.0 release as we bumped the aws sdk gem up a large number of releases.
